Cl 667 rubocop on ci phase 4

koendehondt opened the pull request:
## Changes

These offences were fixed:  Style/MapToHash, Performance/Sum, Layout/BlockEndNewline, Layout/ElseAlignment, Layout/EndAlignment.

## How urgent is a code review?

Not urgent. As soon as this PR is merged, another batch of offences can be fixed.

koendehondt committed:
[CL-667] Fix Layout/ElseAlignment and Layout/EndAlignment

koendehondt committed:
[CL-667] Fix Layout/BlockEndNewline

koendehondt committed:
[CL-667] Fix Layout/ArrayAlignment issues after correcting Layout/BlockEndNewline

koendehondt committed:
[CL-667] Merge branch 'master' into CL-667-rubocop-on-ci-phase-4

koendehondt committed:
[CL-667] Merge branch 'master' into CL-667-rubocop-on-ci-phase-4

koendehondt committed:
[CL-667] Fix Rubocop issue after merging master

koendehondt committed:
[CL-667] Fix Performance/Sum offences

koendehondt committed:
[CL-667] Adapt condition after earlier change to use flat_map

Originally user_ids could be nil. In the previous commit it is always an array.

koendehondt committed:
[CL-667] Fix Style/MapToHash offences

koendehondt committed:
[CL-667] Merge branch 'master' into CL-667-rubocop-on-ci-phase-4

sebastienhoorens reviewed (approved):
All looks good. No remarks.

koendehondt committed:
[CL-667] Merge branch 'master' into CL-667-rubocop-on-ci-phase-4

# Conflicts:
#	back/app/controllers/web_api/v1/stats_ideas_controller.rb
#	back/spec/acceptance/stats_ideas_spec.rb

koendehondt committed:
[CL-667] Keep RuboCop happy after merging master
